src/components/TaskDetail.jsx

```diff
@@ -11,10 +11,17 @@ const LABEL_COLORS = [
   '#6f42c1', '#fd7e14', '#20c997', '#e83e8c'
 ];
 
+const PRIORITY_LEVELS = {
+  HIGH: { label: 'High', color: '#dc3545' },
+  MEDIUM: { label: 'Medium', color: '#ffc107' },
+  LOW: { label: 'Low', color: '#28a745' }
+};
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
// Move to src/constants/taskConstants.js
export const PRIORITY_LEVELS = {
  HIGH: { label: 'High', color: '#dc3545' },
  MEDIUM: { label: 'Medium', color: '#ffc107' },
  LOW: { label: 'Low', color: '#28a745' }
};
```
The PRIORITY_LEVELS constant is defined within the component file but represents business logic. Consider moving it to a separate constants file for better maintainability and reusability.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;